src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import MutualExclusivityTable from "./MutualExclusivityTable";
+import {observer} from "mobx-react";
+import {Checkbox} from 'react-bootstrap';
+import styles from "./styles.module.scss";
+import {computed, observable} from "mobx";
+import {MutualExclusivity} from "../../../shared/model/MutualExclusivity";
+import Combinatorics from 'js-combinatorics';
+import fisherTest from 'fishertest';
+
+export interface IMutualExclusivityTabProps {
+    // a mapping from Hugo Gene Symbol to list of booleans,
+    // each element of the list representing the altered status of a sample
+    isSampleAlteredMap: {[key: string]: boolean[]};
+}
+
+export function calculateAssociation(logOddsRatio: number): string {
+    return logOddsRatio > 0 ? "Tendency towards co-occurrence" : "Tendency towards mutual exclusivity";
+}
+
+export function count(valuesA: boolean[], valuesB: boolean[]): [number, number, number, number] {
```

Comment:
this is kind of an important function so maybe a more descriptive name would be helpful.  like, countCoocurrences or something?